### PR TITLE
Add single task get api

### PR DESCRIPTION
**Pull request by faysal-alam-sazim**
### Description of Change
- Added single task get api
### Associated Ticket Link:
- https://trello.com/c/YDwnV55K

### CODE REVIEW

**Comment by ishmam-iqbal-sazim** [comment]
Please rebase this properly.

File: shared/redux/rtk-apis/tasksAPI.ts [hunk 1]
```diff
@@ -44,6 +44,10 @@ export const api = createApi({
         method: "DELETE",
       }),
     }),
+
+    getSingleTask: builder.query<TTask, string>({
+      query: (id) => `todos/:${id}`,
```
Comment (ishmam-iqbal-sazim) [inline shared/redux/rtk-apis/tasksAPI.ts:49]:
```suggestion
      query: (id) => `todos/${id}`,
```

You don't need the colon. It is used in the backend to indicate that the param taken will be dyanmic

**Review by ishmam-iqbal-sazim — APPROVED** [review]
Approved pending comment.

File: shared/redux/rtk-apis/tasksAPI.ts [hunk 2]
```diff
@@ -44,6 +44,10 @@ export const api = createApi({
         method: "DELETE",
       }),
     }),
+
+    getSingleTask: builder.query<TTask, string>({
```
Comment (ragib-hossain-sazim) [inline shared/redux/rtk-apis/tasksAPI.ts:48]: [Nitpick] `getSingleTask` => `getTask`

**Review by ragib-hossain-sazim — APPROVED** [review]
Approved pending comments. Merge the PR after you are done addressing the comment.

**Review by ragib-hossain-sazim — APPROVED** [review]

**Pull request merged**